helm/alfresco-content-services/templates/_helpers-elasticsearch.tpl

```diff
@@ -1,5 +1,8 @@
 {{- define "repo.elasticsearch.config" -}}
 -Dindex.subsystem.name=elasticsearch
+{{- if eq (index .Values "alfresco-elasticsearch-connector" "elasticsearch" "protocol" | default .Values.global.elasticsearch.protocol) "https" }}
+-Delasticsearch.secureComms=https
+{{- end }}
```

Comment:
In addition to this environment variable, `encryption.ssl.truststore.*` values should include also the public certificate from the Elasticsearch server TLS protocol.

Reply:
why is that needed? our images should have proper cacerts and ssl certificates used by aws opensearch endpoints should already be recognized as trusted

Reply:
You're right @gionn 
Repository is using a custom truststore to communicate with SOLR, but for Elasticsearch standard REST Client is used (so default JVM `cacerts` is enough).